cpp/inc/bond/core/reflection.h

```diff
@@ -101,24 +101,21 @@ struct FieldTemplate
     /// @brief Static data member describing field metadata
     static const Metadata& metadata;
 
-    /// @brief Static data member representing the field pointer
-    static const field_pointer field;
-
```

Comment:
If this materially reduces the lib and PDB size, then it's something we can consider. The next release for C++ will have some other breaking changes (albeit minor ones) like this.